gcp-py-functions/README.md

```diff
@@ -48,66 +48,85 @@ You can connect a [Flic](https://flic.io) button using [IFTTT](https://ifttt.com
+1. Set the target destination to compute directions to:
 
      ```bash
-    pulumi config set destination <DESTINATION> --secret
+    pulumi config set destination <DESTINATION>
     ```
 
-1. [Optional] Configure an offset (i.e. it takes me 5 minutes to get down to my car), by running the following:
+1. (Optional) Add a travel time offset, e.g. add 5 minutes to the estimate.
 
     ```bash
-    pulumi config set travelOffset <TRAVEL_OFFSET> --secret
+    pulumi config set travelOffset <TRAVEL_OFFSET>
     ```
 
-### Add in Texting with Twilio
+### Twilio for SMS Notifications
 
-1. Set up your [Twilio](https://www.twilio.com/) account.
+To have the Cloud Function send a text message, you'll need to a Twilio key too:
 
-1. Run the following commands based on your Twilio account:
+1. Log into your [Twilio](https://www.twilio.com/) account, and create a new access token
+   and/or phone number to send SMS messages from.
+
+1. Add the Twilio configuration data to your Pulumi stack:
 
     ```bash
     pulumi config set twillioAccessToken <TWILIO_ACCESS_TOKEN> --secret
     pulumi config set twillioAccountSid <TWILIO_ACCOUNT_SID> --secret
-    pulumi config set fromPhoneNumber <FROM_PHONE_NUMBER> --secret
+    pulumi config set fromPhoneNumber <FROM_PHONE_NUMBER>
     ```
 
-1. Add the number you would like notify by running the following:
+1. Enter the phone number the Cloud Function will send messages to:
 
     ```bash
     pulumi config set toPhoneNumber <TO_PHONE_NUMBER> --secret
     ```
 
-### Set up the Flic Button
+### Flic Button to Trigger the Cloud Function
+
+With Pulumi having setup the cloud infrastructure, the next step is to have a simple way to trigger
+it. With [Flic](https://flic.io) you can trigger the Cloud Function with literally the push
+of a button.
+
+To make sure to include the button presser's location, you can use [IFTTT](https://ifttt.com).
 
-1. To set up the Flic button, install the Flic app on your phone and pair your phone. Enable location services for the Flic app and add an IFTTT for one of the click gestures.
+1. Install the Flic app on your phone and pair your button. Enable location services for the Flic app
+   and add an IFTTT for one of the click gestures.
 
 1. Create a new Applet on IFTTT: "If You click a Flic, then Make a web request"
-    * For "If" select the "Flic" service then " "Flic is clicked"
+    * For "If" select the "Flic" service then "Flic is clicked".
     * Select your Flic button and the appropriate gesture from the menu.
     * For "Then" select the "Make a web request" service
-    * Under URL enter following (replace <FUNCTION_URL> with the value from `pulumi stack output fxn_url`): `<FUNCTION_URL>?long={{Longitude}}&lat={{Latitude}}`
+    * Under URL enter following (replace `<FUNCTION_URL>` with the value from `pulumi stack output fxn_url`): `<FUNCTION_URL>?long={{Longitude}}&lat={{Latitude}}`
```

Comment:
I realize I never added instructions to run `pulumi up` after setting configuration. I can add that once we merge this into my PR.